bedrock/mozorg/templates/mozorg/about/leadership.html

```diff
@@ -1141,6 +1141,12 @@ <h3 class="group-title" id="board-corp">{{ _('Mozilla Corporation') }}</h3>
       </li>
       <li class="vcard" itemscope itemtype="http://schema.org/Person">
         <figure class="headshot">
+          <img class="photo" itemprop="image" alt="" src="{{ static('img/mozorg/about/leadership/julie-hanna.jpg') }}">
```

Comment:
Yup, it should. Seems I had a few headshots that hadn't gotten the high-res treatment because they were placeholders until the last minute, but we have high-res pics for those folks now. Updated this one and the others (still only have small pics for some people).